### ISSUE TITLE
How to use getMessage method from bot to reply_to_message_id bot?

### ISSUE THREAD

**Issue by azkadev** [tdlib/td#2475]
i know bots can't read bot messages, but in telegram bot api i made a jsondump command and replied to the bot message it appears, but in tdlib i can't just this error appears

Result:
```json
{
    "@type": "error",
    "code": 404,
    "message": "Not Found",
}
```

**Comment by AYMENJD** [comment]
Use `getRepliedMessage`.

**Comment by azkadev** [comment]
> Use `getRepliedMessage`.

I have tried using that method but the result is still the same

**Comment by azkadev** [comment]
yes I have filled in all the parameters needed but it doesn't work if the reply to another bot if the normal user is successful,

**Comment by levlam** [comment]
Bot API server uses `getRepliedMessage` to fetch the replied message in supergroups, even it is inaccessible otherwise to the bot.
